### PR TITLE
Unhide translations widgets in the django admin

### LINKED ISSUE #8450: Handle translated fields in django admin
Translated fields currently do not show up in the django admin - for instance if you look at the django admin page for an add-on, it should show the name, but doesn't.

Even if it did show up, we need to display all translations in the change pages instead of just the translation in the user's current language. We've got some form fields for this, but they need to be integrated into the admin.

### PR DESCRIPTION
**Pull request by diox**
Also add a bit of custom css/js to make them easier to edit.

It turns out that just making sure it has a widget by default before rendering fixes the `is_hidden` property which then fixes displaying of the widget in the admin. The model field already had a custom `formfield()` returning corresponding widget in model forms, and therefore the admin.

Screenshots:
**Before:**
Before
**After:**
After

Fix #8450

### CODE REVIEW

File: src/olympia/translations/widgets.py [hunk 1]
```diff
@@ -101,12 +104,16 @@ def delete_locale(s):
         return rv
 
     def format_output(self, widgets):
-        s = super(TransMulti, self).format_output(widgets)
+        # Gather output for all widgets as normal...
+        rendered_widgets = super(TransMulti, self).format_output(widgets)
+        # ...But also add a widget that'll be cloned for when we want to add
+        # a new translation. Hide it by default.
```
Comment (eviljeff) [inline src/olympia/translations/widgets.py]: How does the page allow new translations to be added?  It wasn't obvious from the screenshot (and nothing in your .js so I assume it's built-in functionality?)

Comment (diox) [reply src/olympia/translations/widgets.py]: It's not possible in the admin - only possible through devhub (which uses a different bit of javascript) - I thought it wasn't as useful in the admin.

Comment (eviljeff) [reply src/olympia/translations/widgets.py]: I agree about it not being that useful - from reading the comment I wasn't sure if admin _did_ allow you to do it though.  Nit on comment, then.

Comment (diox) [reply src/olympia/translations/widgets.py]: Tweaked the comment in https://github.com/mozilla/addons-server/pull/8465/commits/ca9a7c706c1ef1bc2331ef48f59f7879e8818b0e

**Pull request merged**